### PR TITLE
Feature/jdf/add logo

### PR DESCRIPTION
**Pull request by jfrankl**
Chip asked me to add in the logo.

image

image

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -57,7 +56,12 @@
             <a href="#menu" class="menu-link"><i class="icon-menu"></i></a>
             <div id="card-holder">
             </div>
+            <div id="confirmation" class="quest-confirmation clearfix">
```
Comment (caseypt) [inline src/index.html]: Looks like this snuck in from the other PR.

**Comment by caseypt** [comment]
Just to be clear, the new style for the button should only appear when the map is active, correct?

**Comment by jfrankl** [comment]
What button? Sorry, I haven't thought about this in a while.

On Tue, Jun 23, 2015 at 3:07 PM, Casey Thomas <notifications@github.com>
wrote:

> Just to be clear, the new style for the button should only appear when the
> map is active, correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/pwd-waterworks-revealed/pull/93#issuecomment-114612926>
> .
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

**Comment by caseypt** [comment]
The app menu button.

Map is active:

selection_027

Map is inactive:

selection_028

**Comment by jfrankl** [comment]
Yep! Intentional.

On Tue, Jun 23, 2015 at 3:11 PM, Casey Thomas <notifications@github.com>
wrote:

> The app menu button.
>
> Map is active:
>
> [image: selection_027]
> <https://cloud.githubusercontent.com/assets/1042475/8314835/1d967882-19ba-11e5-8278-7503f355f416.png>
>
> Map is inactive:
>
> [image: selection_028]
> <https://cloud.githubusercontent.com/assets/1042475/8314837/27b96e64-19ba-11e5-82d1-ddd4fca84898.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/pwd-waterworks-revealed/pull/93#issuecomment-114613838>
> .
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

**Comment by caseypt** [comment]
+1 besides the confirmation element.

**Pull request merged**